### PR TITLE
DFc-12706-updated response headers with 3 new names ExpectCT, Cross d…

### PR DESCRIPTION
**Pull request by vallabha24**
…omain and feature policy

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/addons_registry.sf [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+  },
+  {
+    "AddonName": "TestJobProfiles",
+    "Key": "1937ed5a-1397-47f8-918c-9e89f6e0bc59",
+    "IsDeleted": false
+  },
+  {
+    "AddonName": "WorkingHPE",
+    "Key": "b3b7ed98-8ee3-4d92-87d5-6653558d3274",
+    "IsDeleted": true
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/addons_registry.sf:15]: why is this changed for this scope?

**Comment by vallabha24** [comment]
No need of addon. so creating another PR
